src/main/webapp/documentManager/showDocument.jsp

```diff
@@ -146,6 +150,8 @@
             String currentDate = new SimpleDateFormat("yyyy-MM-dd").format(new Date());
 
             Integer docCurrentFiledQueue = null;
+
+            request.setAttribute("mrpProviderName", mrpProviderName);
```

Comment:
why does this variable need to be set in the request scope?

Reply:
In this case it would be better to keep the request scope clean by inserting mrpProviderName into the page scope.